jdi-light-bootstrap-tests/src/test/java/io/github/epam/testng/TestNGListener.java

```diff
@@ -40,7 +40,7 @@ public void afterInvocation(IInvokedMethod method, ITestResult r) {
             String result = getTestResult(r);
             logger.step("=== Test '%s' %s [%s] ===", TEST_NAME.get(), result,
                     new SimpleDateFormat("mm:ss.SS").format(new Date(currentTimeMillis()-start.get())));
-            if (result.equals("FAILED")) {
+            if (("FAILED").equals(result)) {
```

Comment:
It is not necessary to put string in parentheses. You should write:
if ("FAILED".equals(result)) {